src/main/kotlin/graphics/scenery/backends/Renderer.kt

```diff
@@ -12,6 +10,7 @@ import kotlinx.coroutines.*
 import kotlinx.coroutines.channels.Channel
 import kotlinx.coroutines.selects.select
 import org.lwjgl.system.Platform
+import java.lang.RuntimeException
```

Comment:
This is weird... does kotlin really make you import `java.lang` packages?

Reply:
It does, but only because this was not using Kotlin's version of it. It was unnecessary and I've removed it.